website/content/guides/1.7-template-upgrade.mdx

```diff
@@ -0,0 +1,154 @@
+---
+page_title: Upgrading your template to use Packer init
+sidebar_title: Upgrade Your Template to use Packer init
+---
+
+# Upgrading your template to be compatible with `packer init`
+
+The `packer init` command is introduced in version v1.7.0 at the same time the [Packer Plugin SDK](https://github.com/hashicorp/packer-plugin-sdk)
+is officially released. The Packer Plugin SDK is a change that benefits plugins maintainers, whilst the `init` command
+benefits Packer users by providing a better control of plugin usage within a Packer template configuration.
```

Comment:
```suggestion
The `packer init` command introduced in version v1.7.0 benefits Packer users by providing better control of plugin usage within a Packer template configuration.
```
I think we can drop the call out for the SDK here as it release time is not relevant for this guide. The call out you have below `"The easiest way to know if the plugin you use fits those requirements, is by looking for SDK migration in their CHANGELOG"` is perfect and helps make this point.

Reply:
Great! If you felt that was enough then you right. I wrote this part before right the other one.